### PR TITLE
fix: Allow pasting a big workflow

### PR DESCRIPTION
**Pull request by mutdmour**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

File: packages/editor-ui/src/components/Node.vue [hunk 1]
```diff
@@ -502,7 +502,9 @@ export default defineComponent({
 		}
 	},
 	mounted() {
-		this.setSubtitle();
+		setTimeout(() => {
```
Comment (netroy) [inline packages/editor-ui/src/components/Node.vue:505]: shouldn't we do this in the `isActive` watcher as well? and if yes, wouldn't it make sense to move the `setTimeout` inside `setSubtitle` ?

Comment (mutdmour) [reply packages/editor-ui/src/components/Node.vue:505]: no settimeout is only needed here when a lot of nodes are mounted at the same time (pasting a big workflow for example)..

Comment (netroy) [reply packages/editor-ui/src/components/Node.vue:505]: previously `this.$emit('run')` happened after `setSubtitle` finished. are we certain that breaking this sequence isn't going to cause any issues?

Comment (mutdmour) [reply packages/editor-ui/src/components/Node.vue:505]: yes, it should be fine.. that's related to showing success state for nodes in executions view

Comment (alexgrozav) [reply packages/editor-ui/src/components/Node.vue:505]: I think `this.$nextTick(() => {})` be more appropriate here.

**Review by alexgrozav — APPROVED** [review]

File: packages/editor-ui/src/components/Node.vue (hunk 1 above)
Comment (mutdmour) [reply packages/editor-ui/src/components/Node.vue:505]: hmm I have run into problems in the past with this.. that's why i default to `setTimeout` it always works

Comment (alexgrozav) [reply packages/editor-ui/src/components/Node.vue:505]: Ok, no strong opinion for it. If it works it works.

Comment (mutdmour) [reply packages/editor-ui/src/components/Node.vue:505]: just tested `nextTick` and it works here.. but will stick with `setTimeout` if you don't mind then

**Pull request merged**